consts/scrcmd.json

```diff
@@ -8,6 +8,19 @@
                 "SAVE_TYPE_FULL_SAVE",
                 "SAVE_TYPE_QUICK_SAVE"
             ]
+        },
+        "@BadgeID": {
+            "type": "enum",
+            "values": [
+                "BADGE_ID_COAL",
+                "BADGE_ID_FOREST",
+                "BADGE_ID_COBBLE",
+                "BADGE_ID_FEN",
+                "BADGE_ID_RELIC",
+                "BADGE_ID_MINE",
+                "BADGE_ID_ICICLE",
+                "BADGE_ID_BEACON"
```

Comment:
**quibble**: I wonder if it's reasonable to provide aliases for these. I have a hard time remembering which is which. 😅 

**chore**: Add a `MAX_BADGES` or somesuch to this enum.

Reply:
Would you mind giving me an example of what you mean by alias here?